src/components/KeyboardShortcutRow.jsx

```diff
@@ -38,7 +38,7 @@ function KeyboardShortcutRow({
         <span
           {...css(styles.KeyboardShortcutRow_key)}
           role="img"
-          aria-label={label}
+          aria-label={`${label},`}
```

Comment:
?

Reply:
https://github.com/airbnb/react-dates/pull/825#discussion_r148938046

Reply:
I can add a comment to the code on this as well.